src/App.tsx

```diff
@@ -15,71 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface Props{
+  goodsList: string[];
+}
```

Comment:
Why do you need this type of `Props` here? It could be `{}`, but why  this one 👇